### PR TITLE
do not write to stderr

### PR DESCRIPTION
**Pull request by nippysaurus**
Depending on what sort of application is using this library, STDOUT and
STDERR can be pretty sacred spaces, and can be very strict about the
rules surrounding formatting.

### CODE REVIEW

File: lib/ruby-rtf/parser.rb [hunk 1]
```diff
@@ -283,7 +283,6 @@ def handle_control(name, val, src, current_pos)
       else
         unless @seen[name]
           @seen[name] = true
-          STDERR.puts "Unknown control #{name.inspect} with #{val} at #{current_pos}"
```
Comment (dj2) [inline lib/ruby-rtf/parser.rb]: Should we add some kind of logger or something? Or maybe add a debug flag and enable this in debug?

**Comment by nippysaurus** [comment]
@dj2 I've updated to allow the user more control over the error message.

I'm just not sure if `unknown_control_warning_enabled` is a little too wordy?

Also using `warn` [allows the user more control over the output](https://apidock.com/ruby/Kernel/warn).
